assets/app/features/upgrade/controllers/upgrade-administration-repositories-checks.controller.js

```diff
@@ -52,27 +54,22 @@
                 .then(
                     // In case of success
                     function (repoChecksResponse) {
-
-                        _.forEach(repoChecksResponse.data, function(value, key) {
-                            vm.repoChecks.checks[key].status = value;
+                        // Iterate over our map
+                        _.forEach(ADMIN_REPO_CHECKS_MAP, function (repos, type) {
+                            _.forEach(repos, function(repo) {
+                                vm.repoChecks.checks[repo].status = repoChecksResponse.data[type]['available']
```

Comment:
Nope, sorry, that is python style applied to javascript, I'll fix it to use `.available`